content/podcast/013-ryan.md

```diff
@@ -0,0 +1,400 @@
+
+*James Munns*
+
+Yeah… I think you really nailed it on how wide embedded is like embedded spans everything from like I'm writing assembly on ah an 8 bit micro all the way up to like, embedded Linux deployment and we call all of that embedded. Yeah, I did a podcast episode a couple years ago now with Francois from interrupt who used to be at Pebble and we talked about this a lot of like —you talk about how people got into it where, especially on the lower end of when you're talking about like your PIC8s or whatever like, a lot of people were like double Es who were they were the one that was like software inclined, and they figured out how to get the PID Loop running on their PIC so they could have a better motor controller or something like that and they came up from that. Or you get the people from the opposite side who were doing backend services and they had done a little bit of electronic stuff before, so they got roped in to writing the microcontroller code. And you really do have these 2 very different worlds that are smashing together in the middle, where you get this really weird gray area of everything from like 32 bit microcontrollers up to very small microprocessors. There is no one style and there's no one right answer and particularly when everyone's cost optimizing or trying to get the most hard realtime out of everything that they're doing, there's no one size fit all approach; which is, whew, it's something for sure.
```

Comment:
I don't know exactly what interrupt refers to (if it should also be in the links) [here? ](https://interrupt.memfault.com/)
PIC8 
double Es

Reply:
Yep, that interrupt.

PIC8 is a brand of microcontrollers: https://www.microchip.com/en-us/products/microcontrollers-and-microprocessors/8-bit-mcus

double-E or "EE": "Electrical Engineer(ing)"